### PR TITLE
Implemented new feature: display last error log in scenario description

### PR DESCRIPTION
**Pull request by vrymar**
- Implemented new feature: display last error log in scenario description
- Implemented unit tests for the new feature
- Improved dependencies references in build.gradle

### CODE REVIEW

**Comment by vrymar** [comment]
@HardNorth could you please review this PR?

**Comment by HardNorth** [comment]
@vrymar Sure, will do this tomorrow.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 ### Removed
 - Shutdown hook register on supplied Launch, by @HardNorth
 
+## [5.0.4]
```
[the remark this answers is not in the document]
Comment (vrymar) [reply CHANGELOG.md:7]: Should I put it under [Unreleased] ?
E.g.:
## [Unreleased]
### Added
- Implemented new feature: display last error log in scenario description, by @vrymar
- Implemented unit tests for the new feature, by @vrymar
### Changed
- Improved dependencies references in build.gradle, by @vrymar

File: src/main/java/com/epam/reportportal/karate/ReportPortalUtils.java [hunk 2]
```diff
@@ -294,35 +295,56 @@ public static TestCaseIdEntry getTestCaseId(@Nonnull Scenario scenario) {
...
+		Scenario scenario = result.getScenario();
+		FinishTestItemRQ rq = buildFinishTestItemRq(Calendar.getInstance().getTime(), result.getFailureMessageForDisplay() == null ? ItemStatus.PASSED : ItemStatus.FAILED);
+		rq.setDescription(buildDescription(scenario, result.getErrorMessage(), getParameters(scenario)));
+		return rq;
+	}
+
+	private static String buildDescription(Scenario scenario, String errorMessage, List<ParameterResource> parameters) {
+		StringBuilder descriptionBuilder = new StringBuilder();
+
+		if (parameters != null && !parameters.isEmpty()) {
```
[the remark this answers is not in the document]
Comment (vrymar) [reply src/main/java/com/epam/reportportal/karate/ReportPortalUtils.java:330]: But this one may return null: https://github.com/reportportal/agent-java-karate/blob/7710c7fe5fbd37272ab7b1061543a27f9ce40581/src/main/java/com/epam/reportportal/karate/ReportPortalUtils.java#L279

[the remark this answers is not in the document]
Comment (HardNorth) [reply src/main/java/com/epam/reportportal/karate/ReportPortalUtils.java:330]: Ok, missed that.

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HardNorth) [reply CHANGELOG.md:7]: @vrymar Yes, exactly.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,6 @@
 # Changelog
 
 ## [Unreleased]
-### Removed
-- Shutdown hook register on supplied Launch, by @HardNorth
```
Comment (HardNorth) [inline CHANGELOG.md:5]: Please don't revert my changes.

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Review by HardNorth — APPROVED** [review]

**Pull request merged**